### PR TITLE
Set `is_jmx` in solr template

### PR DESCRIPTION
**Pull request by albertvaka**
Fixes: #790

### CODE REVIEW

File: recipes/solr.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 include_recipe 'datadog::dd-agent'
 
 # Monitor solr
-# @see https://github.com/DataDog/integrations-core/blob/master/solr/conf.yaml.example Solr Example
+# @see https://github.com/DataDog/chef-datadog/blob/master/templates/default/solr.yaml.erb Solr Example
```
Comment (mx-psi) [inline recipes/solr.rb:4]: Maybe
```suggestion
# @see https://github.com/DataDog/integrations-core/blob/master/solr/datadog_checks/solr/data/conf.yaml.example Solr Example
```
?

Comment (albertvaka) [reply recipes/solr.rb:4]: I totally copied the URL from the wrong tab 😂

**Review by mx-psi — APPROVED** [review]

**Review by RulerOf — APPROVED** [review]
Changes working on my test kitchen

**Comment by albertvaka** [comment]
Thanks for testing this @RulerOf :)

**Pull request merged**